run.sh

```diff
@@ -69,14 +57,6 @@ prepare () {
   SOURCE_DATA_DIR="./data"
 }
 
-execute_raw_data_qa() {
```

Comment:
`execute_raw_data_qa` appears to be depreciated, and it was relying on `DATA_SOURCES_DIR` (the covid-data-public directory) so I've opted to just remove it from this script. I didn't remove the code itself in `check_raw_case_death_data.py`

Reply:
👍  I think we'd be fine to remove the source code too if we ever feel like it.  It was an early attempt at better QA of our data but didn't really pan out.

Reply:
cool, I've gone ahead and removed it in 8c3fa830bd62d9a0b78b52fce8e2c42d29f5d963 (along with updating .github/workflows/CI_test_deploy_api.yml which I missed previously)